simulation.py

```diff
@@ -78,20 +79,42 @@ def extract_data(raw_data, data_list, price_dataset, region):
             name : raw_data[region][name]
             for name in data_list if name in raw_data[region]}}
 
-def handle_result(
-        results, eval_results, alpha_path,
-        quiet_mode, output_path):
+def out_put_on_cmd(results, instruments):
+    """
+    Change results format to have a good outlook on command line.
+    """
+    form = {
+        'instrument': [],
+        'date': [],
+        'pnl': [],
+        'cost': [],
+        'position': []
+    }
+    for today in results:
+        date = today.date
+        for instrument in instruments:
+            form['date'].append(date)
+            form['instrument'].append(instrument)
+            form['pnl'].append(today.pnl[instrument])
+            form['cost'].append(today.cost[instrument])
+            form['position'].append(
+                today.position_raw.get(instrument, 0.))
+    return pandas.DataFrame(data=form)
+
+def handle_result(alpha_path, results_set, quiet_mode, output_path):
     """
     Handle results of simulation.
     """
+    results, eval_results, instruments = results_set
     if output_path:
         output_data = {
             'simulation_results': results,
             'evaluation_results': eval_results}
         with open(os.path.join(output_path, 'results'), 'wb') as file:
             pickle.dump(output_data, file)
     if not quiet_mode:
-        print(alpha_path, results, eval_results, sep='\n')
+        print(alpha_path, out_put_on_cmd(results, instruments),
```

Comment:
I found that this will print out only heads and tails of the data frame. Is it possible to print out all of them?